pkg/routeagent/controllers/route/route.go

```diff
@@ -380,7 +383,7 @@ func (r *Controller) processNextPod() bool {
 		}
 
 		r.podWorkqueue.Forget(obj)
-		klog.V(4).Infof("Pod event processed by route controller")
+		klog.V(log.DEBUG).Infof("Pod event processed by route controller")
```

Comment:
I suggest making this Info as well so we can see when it finished processing and the elapsed time:

`klog.Infof("Successfully processed submariner-route-agent pod %q", name)`

Reply:
I think it's better as debug, if we want to check elapsed times, etc, which should be very low. Although I will change the formatting.

Reply:
Done, but as DEBUG